apps/frontend/src/libs/components/input/styles.module.css

```
@@ -0,0 +1,53 @@
+.input-wrapper {
+	display: flex;
+	flex-direction: column;
+}
+
+.input-label {
+	font-size: 16px;
+}
+
+.input-container {
+	position: relative;
+}
+
+.input__field {
```

```suggestion
.input-field {
```
Let's have all style names this way